osc-export/README.md

```diff
@@ -12,4 +12,12 @@ Locally debugging the server is easier using `server-debug.bndrun`.
 * Both `server.bndrun` and `server-debug.bndrun` can be used with the Eclipse Java debugger.
 
 ### Dependencies
-This project will list all dependencies needed to create an executable JAR of OSC.
\ No newline at end of file
+This project will list all dependencies needed to create an executable JAR of OSC.
+
+### Merging the two old truststores
+The merging was performed with the following command:
+
+ `keytool -importkeystore -srcstoretype JKS -deststoretype JKS -srckeystore vmidcKeyStore.jks -destkeystore osctrustore.jks`
+
+ For additional info, see [keytool documentation](https://docs.oracle.com/javase/6/docs/technotes/tools/windows/keytool.html),
```

Comment:
it is not necessary to include this information in the readme here.

We should add instructions in the community github for linux and windows(less important since this not our deployment platform).

For linux if the upgrade script is getting fixed to merge the keystores, then it maynot be necessary to add any information.